### PR TITLE
Add international airports at z8-9

### LINKED ISSUE #1082: Change aerodrome_label min zoom level to 7
Hello!

Would it be possible to change the min zoom level limit on the [aerodrome_label](https://github.com/openmaptiles/openmaptiles/blob/master/layers/aerodrome_label/aerodrome_label.sql#L40) for example to 7?

Currently, it is not possible to create a map where we could show the locations of the major airports on for example US State level.

I can make the changes and PR, if we can move forward with this.

### PR DESCRIPTION
**Pull request by ZeLonewolf**
Closes #1082

This PR moves the aerodrome/aerodrome_type consolidation (via template) into the lower-level `osm_aerodrome_label_point` table, and creates a partial index on class `international` for airports with an `iata` code.  The presence of an `iata` code is more significant than an `icao` code which exists at many smaller, general aviation airport.  Therefore, `international` + `iata` should be a sufficient filter.

Zoom 8 international airports in the NYC area (blue dots):
image

Zoom 10 in the same area:
image

### CODE REVIEW

File: layers/aerodrome_label/aerodrome_label.sql [hunk 1]
```diff
@@ -27,17 +27,34 @@ SELECT
...
+    COALESCE(NULLIF(name_en, ''), name) AS name_en,
+    COALESCE(NULLIF(name_de, ''), name, name_en) AS name_de,
+    tags,
+    aerodrome_type AS class,
+    NULLIF(iata, '') AS iata,
+    NULLIF(icao, '') AS icao,
+    substring(ele FROM E'^(-?\\d+)(\\D|$)')::int AS ele,
+    round(substring(ele FROM E'^(-?\\d+)(\\D|$)')::int * 3.2808399)::int AS ele_ft
+FROM osm_aerodrome_label_point
+WHERE aerodrome_type = 'international' AND iata <> '' AND geometry && bbox
```
Comment (TomPohys) [inline layers/aerodrome_label/aerodrome_label.sql:55]: Could you please move the `AND` on separate lines? Like in https://github.com/openmaptiles/openmaptiles/blob/76d5d753d8fae35e4d316f5532461835a3acf36b/layers/place/place.sql#L78

File: layers/aerodrome_label/aerodrome_label.sql [hunk 2]
```diff
@@ -27,17 +27,34 @@ SELECT
...
     NULLIF(icao, '') AS icao,
     substring(ele FROM E'^(-?\\d+)(\\D|$)')::int AS ele,
     round(substring(ele FROM E'^(-?\\d+)(\\D|$)')::int * 3.2808399)::int AS ele_ft
 FROM osm_aerodrome_label_point
 WHERE geometry && bbox
-  AND zoom_level >= 10;
+  AND zoom_level >= 10
+
+UNION ALL
+SELECT
```
Comment (TomPohys) [inline layers/aerodrome_label/aerodrome_label.sql:40]: Could you please move zoom 8 and 9 before zoom 10? Thank you

**Comment by TomPohys** [comment]
Hi @ZeLonewolf, thank you for this PR, it looks great. I write just some styling comments, for keeping code consistency. Thanks

File: layers/aerodrome_label/aerodrome_label.sql (hunk 1 above)
Comment (ZeLonewolf) [reply layers/aerodrome_label/aerodrome_label.sql]: Done!

File: layers/aerodrome_label/aerodrome_label.sql (hunk 2 above)
Comment (ZeLonewolf) [reply layers/aerodrome_label/aerodrome_label.sql:44]: Done!

**Review by TomPohys — APPROVED** [review]

**Comment by TomPohys** [comment]
Awesome! Thanks

File: layers/aerodrome_label/aerodrome_label.sql [hunk 3]
```diff
@@ -19,6 +19,28 @@ CREATE OR REPLACE FUNCTION layer_aerodrome_label(bbox geometry,
             )
 AS
 $$
+SELECT
+    -- etldoc: osm_aerodrome_label_point -> layer_aerodrome_label:z8
+    -- etldoc: osm_aerodrome_label_point -> layer_aerodrome_label:z9
```
Comment (zstadler) [inline layers/aerodrome_label/aerodrome_label.sql:24]: The [integrity check](https://github.com/openmaptiles/openmaptiles/pull/1187/checks?check_run_id=3398934444#step:5:11) said:

```
Warning: node layer_aerodrome_label, port z8 unrecognized
Warning: node layer_aerodrome_label, port z9 unrecognized
```

As a result, the new labels are not shown in [layers/aerodrome_label/etl_diagram.png](https://github.com/ZeLonewolf/openmaptiles/blob/1425f92ad3811624bdea4442ef0a1245b9fa3214/layers/aerodrome_label/etl_diagram.png)

Comment (ZeLonewolf) [reply layers/aerodrome_label/aerodrome_label.sql:24]: Fixed in #1223

**Pull request merged**